pkg/cmd/project/item-edit/item_edit.go

```diff
@@ -108,63 +128,33 @@ func NewCmdEditItem(f *cmdutil.Factory, runF func(config editItemConfig) error)
 	editItemCmd.Flags().StringVar(&opts.date, "date", "", "Date value for the field (YYYY-MM-DD)")
 	editItemCmd.Flags().StringVar(&opts.singleSelectOptionID, "single-select-option-id", "", "ID of the single select option value to set on the field")
 	editItemCmd.Flags().StringVar(&opts.iterationID, "iteration-id", "", "ID of the iteration value to set on the field")
+	editItemCmd.Flags().BoolVar(&opts.clear, "clear", false, "Remove field value")
 
 	_ = editItemCmd.MarkFlagRequired("id")
 
 	return editItemCmd
 }
 
 func runEditItem(config editItemConfig) error {
-	// update draft issue
-	if config.opts.title != "" || config.opts.body != "" {
-		if !strings.HasPrefix(config.opts.itemID, "DI_") {
-			return cmdutil.FlagErrorf("ID must be the ID of the draft issue content which is prefixed with `DI_`")
-		}
-
-		query, variables := buildEditDraftIssue(config)
-
-		err := config.client.Mutate("EditDraftIssueItem", query, variables)
-		if err != nil {
+	// when clear flag is used, remove value set to the corresponding field ID
+	if config.opts.clear {
+		if err := clearItemFieldValue(config); err != nil {
 			return err
 		}
+	}
```

Comment:
`runEditItem` will throw an error at the end if the earlier action logic doesn't return when complete, so let's bubble this up instead:

```suggestion
	// when clear flag is used, remove value set to the corresponding field ID
	if config.opts.clear {
		return clearItemFieldValue(config)
	}
```

Reply:
I'll mark this as resolved as it's fixed in https://github.com/cli/cli/pull/8024/commits/c8b465269e0c2bc800e81c91916681660c7efee1.